fashion-messaging-app-ui/src/App.jsx

```diff
@@ -9,16 +9,13 @@ import LoginForm from './components/LoginForm/LoginForm';
 import SignupForm from './components/SignupForm/SignupForm';
 import Homepage from './components/Homepage/Homepage';
 
-
 const API_KEY = "06ZcwjgbmJBM8T2TxLUZ5iwdXXGxiAgz0Z018b7QPKwR1ExipkFjaAuw";
 const clientAPI = createClient(API_KEY);
 const defaultQuery = "Fashion";
-const PHOTOS = 100;
+const PHOTOS = 1;
```

Comment:
I reduced the amount of photos because the second API I am using has limited calls and since I am still testing if it works, I don't want to use up all the API calls